turbinia/workers/bulk_extractor_test.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for the Bulk Extractor job."""
+
+from __future__ import unicode_literals
+
+import unittest
+import mock
+
+from turbinia.evidence import RawDisk
+from turbinia.workers import bulk_extractor
+from turbinia.workers.workers_test import TestTurbiniaTaskBase
+from turbinia.workers import TurbiniaTaskResult
+
+
+class BulkExtractorTaskTest(TestTurbiniaTaskBase):
+  """Tests for BulkExtractorTask."""
+
+  def setUp(self):
+    # pylint: disable=arguments-differ
+    super(BulkExtractorTaskTest, self).setUp(
+        task_class=bulk_extractor.BulkExtractorTask, evidence_class=RawDisk)
```

Comment:
Maybe change the evidence class?

```suggestion
        task_class=bulk_extractor.BulkExtractorTask, evidence_class=CompressedDirectory)
```